spec/models/pile_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Pile do
+  let(:pile_size) { 3 }
+  let(:pile) { Pile.new Array.new(pile_size).map { Card.random_card } }
+  let(:in_pile) { pile.sample }
+  let(:not_in_pile) { Card.new(face: '0', suit: 'h') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.